src/hooks/screen/useChatInfoHook.ts

```diff
@@ -38,13 +37,12 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
   const initChatInfoData = async () => {
     if (localDb) {
       setLoadingChannelInfo(true);
-      const myId = await getUserId();
-      const myAnonymousId = await getAnonymousUserId();
+
       const data = await ChannelList.getChannelInfo(
         localDb,
         selectedChannel?.id,
-        myId,
-        myAnonymousId
+        signedProfileId,
+        anonProfileId
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is an improvement as it removes the need for calling `getUserId()` and `getAnonymousUserId()` functions directly within `initChatInfoData()`. Instead, it uses `signedProfileId` and `anonProfileId` which are presumably fetched elsewhere in the code and passed into this function. This reduces the number of async operations within this function, potentially improving performance.

```diff
-      const myId = await getUserId();
-      const myAnonymousId = await getAnonymousUserId();
-       const data = await ChannelList.getChannelInfo(
-         localDb,
-         selectedChannel?.id,
-        myId,
-        myAnonymousId
-       );
+      const data = await ChannelList.getChannelInfo(
+        localDb,
+        selectedChannel?.id,
+        signedProfileId,
+        anonProfileId
+      );
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->